src/mmw/apps/modeling/tasks.py

```diff
@@ -56,7 +56,7 @@ def start_histograms_job(polygons):
     }
 
 
-@shared_task(bind=True, default_retry_delay=1, max_retries=20)
+@shared_task(bind=True, default_retry_delay=1, max_retries=42)
```

Comment:
Since the polling interval on the frontend is 500 and timeout is 45,000, it seems that there could be up to 90 polls. So, why is max_retries set to 42?